tools/ipu_train_example.sh

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
It's better to move this script to the `demo` folder, because this is an example instead of a tool.